### PR TITLE
feature: App Sign 添加java版本实现

### PR DESCRIPTION
**Pull request by cctyl**
App Sign 添加java版本实现

### CODE REVIEW

File: docs/misc/sign/APP.md [hunk 1]
```diff
@@ -63,3 +63,71 @@ print(query)
...
+        for (Map.Entry<String, String> entry : sortedParams.entrySet()) {
+            if (queryBuilder.length() > 0) {
+                queryBuilder.append('&');
+            }
+            queryBuilder
+                    .append(URLEncoder.encode(entry.getKey(), StandardCharsets.UTF_8))
+                    .append('=')
+                    .append(URLEncoder.encode(entry.getValue(), StandardCharsets.UTF_8));
+        }
+        return generateMD5(queryBuilder.toString() + APP_SEC);
```
Comment (xiaoyv404) [inline docs/misc/sign/APP.md:105]: `.toString()` method is unnecessary

**Review by xiaoyv404 — CHANGES_REQUESTED** [review]

File: docs/misc/sign/APP.md [hunk 2]
```diff
@@ -23,7 +23,7 @@
 
 ## Demo
 
-该 Demo 提供 [Python](#Python) 语言例程
+该 Demo 提供 [Python](#Python)  [Java](#Java) 语言例程
```
Comment (xiaoyv404) [inline docs/misc/sign/APP.md:26]: 请使用 `和` 连接

**Review by xiaoyv404 — CHANGES_REQUESTED** [review]

File: docs/misc/sign/APP.md (hunk 2 above)
Comment (cctyl) [reply docs/misc/sign/APP.md:26]: 这个是我需要修改后，重新提交。还是直接点击resolve conversation

Comment (xiaoyv404) [reply docs/misc/sign/APP.md:26]: 提交，然后[重新请求审核](https://docs.github.com/zh/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews#re-requesting-a-review)

**Review by xiaoyv404 — APPROVED** [review]

**Pull request merged**